orca/server_metrics.go

```diff
@@ -193,6 +195,9 @@ func copyMap(m map[string]float64) map[string]float64 {
 // SetCPUUtilization records a measurement for the CPU utilization metric.
 func (s *serverMetricsRecorder) SetCPUUtilization(val float64) {
 	if val < 0 {
+		if logger.V(2) {
+			logger.Infof("Ignoring CPU Utilization value out of range: %v", val)
```

Comment:
Log valid range here and elsewhere?

Reply:
Ehh, they'll figure it out :)

Reply:
"[0, infy)" this is very few characters to add to logs lol